featurebyte/schema/worker/task/historical_feature_table.py

```diff
@@ -23,6 +23,7 @@ class HistoricalFeatureTableTaskPayload(BaseTaskPayload, HistoricalFeatureTableC
     command: ClassVar[WorkerCommand] = WorkerCommand.HISTORICAL_FEATURE_TABLE_CREATE
     output_collection_name: ClassVar[str] = HistoricalFeatureTableModel.collection_name()
     is_revocable: ClassVar[bool] = True
+    is_rerunnable: ClassVar[bool] = True
```

Comment:
Successful tasks cannot be re-run, is that right?

Reply:
Yes resubmission will only accept tasks that are in a terminal status that is not SUCCESS

Reply:
Yes resubmission will only accept tasks that are in a terminal status that is not SUCCESS